Dockerfile

```diff
@@ -3,13 +3,15 @@ FROM debian:stable
 MAINTAINER Clifton Barnes <clifton.a.barnes@gmail.com>
 
 RUN apt-get update
-RUN apt-get install -y python python-dev python-pip lighttpd
+RUN apt-get install -y python python-dev python-pip python-smbus lighttpd
 RUN pip install flask flask_cors
```

Comment:
I'll add the change to this pull request, but this pull request will then need to be after #44.